eml/App.js

```diff
@@ -20,19 +20,19 @@ import SectionCompleteScreen from './screens/excercise/SectionCompleteScreen';
 import AsyncStorage from '@react-native-async-storage/async-storage';
```

Comment:
all double quotations " should be single quotation ' in all changed files

Reply:
Why?

Reply:
consistency